rust-toolchain.toml

```diff
@@ -0,0 +1,5 @@
+[toolchain]
+channel = "stable"
```

Comment:
My only suggestion is that we set this to our current desired MSRV toolchain, e.g. `"1.90"` or `"1.88"`, or whatever makes the most sense.

This will ensure that building/testing the project from the repo will always use the exact same toolchain, as opposed to whatever `stable` happens to mean at the moment on the persons machine (or in CI)